### PR TITLE
[JEP-226] Improve migration section

### PR DESCRIPTION
**Pull request by stellargo**
Improves the migration section of JEP 226. See https://github.com/jenkinsci/jenkins/pull/4825

### CODE REVIEW

File: jep/226/README.adoc [hunk 1]
```diff
@@ -112,7 +112,15 @@ Implementation wise, this is achieved by plugins ensuring that Fingerprints have
 
 When a plugin is configured and enabled, the fingerprints are transferred from the local storage to the external storage.
 
-This by followed by clearing the local database of fingerprints from Jenkins HOME ONLY if the writes are successful.
+This is followed by clearing the local database of fingerprints from Jenkins HOME ONLY if the writes are successful.
+
+Following new methods are introduced in the `FingerprintStorage` class:
```
Comment (afalko) [inline jep/226/README.adoc:117]:
```suggestion
We have the following new methods in the `FingerprintStorage` class:
```

Comment (stellargo) [reply jep/226/README.adoc]: Proposed new design, so resolving this

**Comment by stellargo** [comment]
@oleg-nenashev @afalko @mikecirioli following our last meeting, would appreciate your reviews here.

**Comment by oleg-nenashev** [comment]
oops, I forgot to merge it

**Comment by oleg-nenashev** [comment]
@stellargo is it still actual after the initial implementation?

**Comment by stellargo** [comment]
@oleg-nenashev the implementation was the same. However I improved the language and made it merge ready :)

**Pull request merged**